de/errors.po

```diff
@@ -1,324 +1,395 @@
 
 msgid "Missing Text"
-msgstr ""
+msgstr "Fehlender Text"
 
 msgid "Please provide some text for language detection."
-msgstr ""
+msgstr "Bitte gib einen Text für die Spracherkennung ein."
 
 msgid ""
 "To translate a message, you *need* a message!\n"
 "Please edit your command to include text."
 msgstr ""
+"Um eine Nachricht zu übersetzen, benötigst du eine Nachricht!\n"
+"Bitte bearbeite deinen Befehl, um Text einzufügen."
 
 msgid "Invalid languages argument"
-msgstr ""
+msgstr "Ungültiges Sprachen-Argument"
 
 msgid "Incorrect format for from/to argument, please edit to fix."
 msgstr ""
+"Falsches Format für Von/Zu-Argument, bitte bearbeiten, um es zu korrigieren."
 
 msgid "Missing Permissions"
-msgstr ""
+msgstr "Fehlende Berechtigungen"
 
 msgid "Please fix that if you're an administrator."
-msgstr ""
+msgstr "Bitte behebe das, wenn du ein Administrator bist."
 
 msgid ""
-"Bot does not have **Manage Roles** or **Manage Nicknames** permissions on "
-"this server."
+"Bot does not have **Manage Roles** or **Manage Nicknames** permissions on this "
+"server."
 msgstr ""
+"Bot hat keine **Rollen verwalten** oder **Nicknamen verwalten** Berechtigungen "
+"auf diesem Server."
 
 msgid ""
-"Bot cannot assign the role **%s** because it is beyond the bot's highest "
-"role. You can fix it by **moving** the bot's primary role (`%s`) higher in "
-"the roles list."
+"Bot cannot assign the role **%s** because it is beyond the bot's highest role. "
+"You can fix it by **moving** the bot's primary role (`%s`) higher in the roles "
+"list."
 msgstr ""
+"Der Bot kann die Rolle **%s** nicht zuweisen, da sie über die höchste Rolle des "
+"Bot hinausgeht. Du kannst es beheben, indem du die primäre Rolle des Bot (`%s`) "
+"in der Rollenliste nach oben verschiebst."
 
 msgid "Invalid Channel"
-msgstr ""
+msgstr "Ungültiger Kanal"
 
 msgid ""
-"The channel %s seems to be either **invalid** or **unreachable** in your "
-"server. Contact an administrator for support."
+"The channel %s seems to be either **invalid** or **unreachable** in your server. "
+"Contact an administrator for support."
 msgstr ""
+"Der Kanal %s in deinem Server scheint entweder **ungültig** oder "
+"**unerreichbar** zu sein. Wenden dich an einen Administrator, um Unterstützung "
+"zu erhalten."
 
 msgid "Invalid Language(s)"
-msgstr ""
+msgstr "Ungültige Sprache(n)"
 
 msgid "Translator could not recognize the following as valid languages:"
 msgstr ""
+"Translator konnte die folgenden Sprachen nicht als gültige Sprachen erkennen:"
 
 msgid "Missing Language"
-msgstr ""
+msgstr "Fehlende Sprache"
 
 msgid ""
-"Please specify a **valid language** for translation (Can be ISO code or "
-"name)."
+"Please specify a **valid language** for translation (Can be ISO code or name)."
 msgstr ""
+"Bitte gib eine **gültige Sprache** für die Übersetzung an (kann ISO-Code oder "
+"ein Name sein)."
 
+# Unklar, was hier genau gemeint ist
 msgid "Explain yourself"
-msgstr ""
+msgstr "Erkläre dich"
```

Comment:
It probably sounds better when we translate it like `What are you doing here`.

So new suggestion would be `Was machst du da`.